docs/dashboard.md

```diff
@@ -0,0 +1,33 @@
+# SC4S Metrics and Events Dashboard
+SC4S Metrics and Events Dashboard enables monitoring crucial metrics and events flow of all the SC4S instances sending data to a chosen Splunk platform.
+
+
+## Functionalities
+
+### Overview metrics
+The dashboard displays the cumulative sum of received and dropped messages of all SC4S instances in chosen interval and for the specified time range. It is 30 seconds span in a 15 minutes window by default.
+
+Received messages panel can be used as a heart beat metric, since a healthy SC4S instance should send at least one message per 30 seconds. This is a metrics message and it's included in the count.
```

Comment:
The Received Messages panel can be used as a heartbeat metric. A healthy SC4S instance should send at least one message per 30 seconds. This metrics message and is included in the count.